theme/layouts/_breadcrumbs.ejs

```diff
@@ -1,6 +1,5 @@
 <%
 const query = site.queryPages;
-if (page.pathname !== "") {
```

Comment:
Those changes are related to breadcrumb, I wonder if we need those?

Reply:
It's just in order to simplify the code. Do you think it's irrelevant in scope of this PR and should I revert it?

Reply:
Ach right, I forgot that we had a special case for Homepage, LGTM.